plugin/builtin/patchtransformer/PatchTransformer_test.go

```diff
@@ -198,6 +199,224 @@ Patch: "something"
 	})
 }
 
+const multipleSMPatchesFile = `
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: myDeploy
+spec:
+  template:
+    spec:
+      containers:
+      - image: public.ecr.aws/nginx/nginx:mainline
+        name: nginx
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: yourDeploy
+spec:
+  template:
+    metadata:
+      labels:
+        new-label: new-value-with-multipleSMPatchesFile
+`
+
+const multipleSMPatchesSuccesfulResult = `
```

Comment:
nit: A neat shorthand if you have multiple `const` is to combine them with parentheses:
```
const (
  multipleSMPatchesFile = `...`
  multipleSMPatchesSuccessfulResult = `...`
)
```

Reply:
fix